src/vonage/video.py

```diff
@@ -163,33 +155,127 @@ def add_stream_to_archive(self, archive_id, stream_id, has_audio=True, has_video
         )
 
     def change_archive_layout(self, archive_id, params=None, **kwargs):
         return self._client.put(
             self._client.video_host(),
             f'/v2/project/{self._client.application_id}/archive/{archive_id}/layout',
             params or kwargs,
-            auth_type=Video.auth_type
+            auth_type=Video.auth_type,
+        )
+
+    def create_sip_call(self, session_id: str, token: str, sip: dict):
+        if 'uri' not in sip:
+            raise SipError('You must specify a uri when creating a SIP call.')
+
+        params = {'sessionId': session_id, 'token': token, 'sip': sip}
+        return self._client.post(
+            self._client.video_host(),
+            f'/v2/project/{self._client.application_id}/dial',
+            params,
+            auth_type=Video.auth_type,
+        )
+
+    def play_dtmf(self, session_id: str, digits: str, connection_id: str = None):
+        if not re.search('^[0-9*#p]+$', digits):
+            raise InvalidInputError('Only digits 0-9, *, #, and "p" are allowed.')
+
+        params = {'digits': digits}
+
+        if connection_id is not None:
+            return self._client.post(
+                self._client.video_host(),
+                f'/v2/project/{self._client.application_id}/session/{session_id}/connection/{connection_id}/play-dtmf',
+                params,
+                auth_type=Video.auth_type,
+            )
+
+        return self._client.post(
+            self._client.video_host(),
+            f'/v2/project/{self._client.application_id}/session/{session_id}/play-dtmf',
+            params,
+            auth_type=Video.auth_type,
+        )
+
+    def list_broadcasts(self, offset=None, count=None, session_id=None):
+        params = {'offset': str(offset), 'count': str(count), 'sessionId': session_id}
+
+        return self._client.get(
+            self._client.video_host(),
+            f'/v2/project/{self._client.application_id}/broadcast',
+            params,
+            auth_type=Video.auth_type,
+        )
+
+    def start_broadcast(self, params: dict):
+        return self._client.post(
+            self._client.video_host(),
+            f'/v2/project/{self._client.application_id}/broadcast',
+            params,
+            auth_type=Video.auth_type,
+        )
+
+    def get_broadcast(self, broadcast_id: str):
+        return self._client.get(
+            self._client.video_host(),
+            f'/v2/project/{self._client.application_id}/broadcast/{broadcast_id}',
+            auth_type=Video.auth_type,
+        )
+
+    def stop_broadcast(self, broadcast_id: str):
+        return self._client.post(
+            self._client.video_host(),
+            f'/v2/project/{self._client.application_id}/broadcast/{broadcast_id}',
+            params={},
+            auth_type=Video.auth_type,
+        )
+
+    def change_broadcast_layout(self, broadcast_id: str, params: dict):
+        return self._client.post(
```

Comment:
Should be `PUT` not `POST`

Reply:
The spec is `POST` but I know you've run this recently and it's `PUT`, so I will change this. Might be worth updating the spec

Reply:
The spec is updated, we were seeing if doing a Draft PR would allow a test deploy but it didn't work. So it's just waiting to be merged into the developer docs.

https://github.com/Vonage/bifrost-dev-docs-mfe/pull/370

Reply:
Gotcha, I was looking at the old api-specification repo